torch/distributed/run.py

```diff
@@ -548,6 +548,16 @@ def get_args_parser() -> ArgumentParser:
         help="Tee std streams into a log file and also to console (see --redirects for format).",
     )
 
+    parser.add_argument(
+        "--filter-ranks",
+        "--filter_ranks",
+        action=env,
+        type=str,
+        default="",
+        help="Only show logs from specified ranks in console (e.g. [--filter-ranks 0 1 2] will "
+        "only show logs from rank 0, 1 and 2). When used with --tee, logs will still be saved to files",
```

Comment:
> When used with --tee, logs will still be saved to files
clarify whether _all_ ranks or just filtered ranks files will be saved to files?

Reply:
I can add another statement to clarify that this only filters outputs to console, not to files. One confusion is that despite being named `tee_stdouts` and `tee_stderrs`, once they are used in torch/distributed/elastic/multiprocessing/__init__.py:252, they are only used by `TailLogs` to output from file to console